package.json

```
@@ -4,14 +4,15 @@
   "description": "",
   "main": "app.js",
   "scripts": {
-    "start": "node app.js",
+    "start": "node --require dotenv/config --inspect=localhost:9229 --experimental-modules app.js",
```

```suggestion
    "start": "node app.js",
    "dev": "node --require dotenv/config --inspect=localhost:9229 --experimental-modules app.js",
```
debugger は別のスクリプトコマンドでしたほうが良きかと